perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -559,6 +574,21 @@ def Prepare(self, vm):
                  vm.db_instance_address)
     logging.info('Complete output is:\n %s', response)
 
+  def _validate_size(self, size):
+    """Validates flag for storage size, throws exception if invalid.
+
+    Args:
+      size: (GB).
+    Returns:
+      size: (GB).
+
+    AWS supports storage sizes from 1GB to 16TB currently.
+    """
+    if size < 1 or size > 16000:
+      sys.exit('Storage size flag given is not valid. Must be between '
```

Comment:
instead of doing sys.exit() can you throw an exception instead? (follow the example of error handling logic in other places of this code).

By throwing exception here, the parent classes of PKB may decide to catch those and initiate a cleanup if necessary, however, doing sys.exit() would void all that possibility.

(As FYI: Currently PKB doesn't usually do cleanup upon benchmark failure, so if a user sees an exception here, they will need to call pkb.py --run_stage=cleanup --run_uri == this run uri, but the key here by throwing an exception up from here, we all the possibility to implement cleanup at the parent class in the future).